mapbox_vector_tile/__init__.py

```diff
@@ -1,9 +1,9 @@
 from mapbox_vector_tile import decoder, encoder
 
 
-def decode(tile, y_coord_down=False):
-    vector_tile = decoder.TileData()
-    message = vector_tile.get_message(tile, y_coord_down)
+def decode(tile, y_coord_down=False, crs_from=None, crs_to=None):
```

Comment:
I think it'd be nice to have this be a little bit more general, and just offer the ability to specify a transform fn: (Point -> Point). This could also mean that we wouldn't need to depend on pyproj, and could just add an example to the readme showing how it could be done.
(Don't mean to take the point -> point part literally, just have it take a (int,int) tuple or 2 parameters)